src/SheetTxt.ts

```diff
@@ -26,8 +27,12 @@ export class SheetTxt {
       const selectedSheet = sheetParams.sheetName
       await Promise.all(sheetParams.cells.map(async cellParams => {
         const cellContents = await this.sheets[sheetParams.spreadsheetId].getSingleCell(selectedSheet, cellParams.cell)
+
+        const dir = dirname(cellParams.path)
+        if (!existsSync(dir)) mkdirSync(dir)
```

Comment:
If we're going to handle making the directory, mkdirSync should probably have recursive:true

At the moment this will fail if eg. path is `a/b/file.txt` and the `a` folder doesn't exist

Reply:
```suggestion
        if (!existsSync(dir)) mkdirSync(dir, { recursive: true })
```